source/_integrations/airgradient.markdown

```diff
@@ -25,6 +25,7 @@ ha_zeroconf: true
 ---
 
 The AirGradient integration will fetch data from your [AirGradient devices](https://www.airgradient.com/).
+AirGradient creates open-source air monitors that measure a wide variety of metrics.
```

Comment:
@airgradienthq do you have any recommendations for a better text? I am looking to implement https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-high-level-description, but I have a slight trouble between making it objective and making it an advertisement (and we want to avoid going 100% ad because then you would have integration pages claiming they are the best and greatest)